python_ta/__init__.py

```diff
@@ -192,6 +190,15 @@ def _check(
                     load_default_config=load_default_config,
                 )
 
+                # Override error messages
+                for error_id, new_msg in messages_config.items():
+                    # Create new message definition object according to configured error messages
+                    message = linter.msgs_store.get_message_definitions(error_id)
```

Comment:
Is this line robust against an `error_id` that isn't present in the store? In case, a logging warning message should be printed, but the message should be skipped over (without crashing).